diff --git a/dist/abis/EntryPoint.abi.js b/dist/abis/EntryPoint.abi.js
index 81d1e2b4cbb6a5439a5db1699008469fbde4aef0..89e298125328a1263dd36fc30b3775cdbf8d9bfa 100644
--- a/dist/abis/EntryPoint.abi.js
+++ b/dist/abis/EntryPoint.abi.js
@@ -564,7 +564,7 @@ export const EntryPointABI = [
             },
             {
                 "name": "commitment",
-                "type": "core::integer::u256",
+                "type": "core::felt252",
                 "kind": "data"
             },
             {
diff --git a/dist/abis/PrivacyPool.abi.js b/dist/abis/PrivacyPool.abi.js
index 3cc5c7a9c38c277055e374b856b4f4a072ef7cb9..4a798a60b4b60300c8d26a79c864a7562940919c 100644
--- a/dist/abis/PrivacyPool.abi.js
+++ b/dist/abis/PrivacyPool.abi.js
@@ -206,7 +206,7 @@ export const PrivacyPoolABI = [
                 "inputs": [],
                 "outputs": [
                     {
-                        "type": "core::integer::u256"
+                        "type": "core::felt252"
                     }
                 ],
                 "state_mutability": "view"
@@ -461,12 +461,12 @@ export const PrivacyPoolABI = [
             },
             {
                 "name": "label",
-                "type": "core::integer::u256",
+                "type": "core::felt252",
                 "kind": "key"
             },
             {
                 "name": "commitment",
-                "type": "core::integer::u256",
+                "type": "core::felt252",
                 "kind": "data"
             },
             {
@@ -476,7 +476,7 @@ export const PrivacyPoolABI = [
             },
             {
                 "name": "preCommitmentHash",
-                "type": "core::integer::u256",
+                "type": "core::felt252",
                 "kind": "data"
             }
         ]
@@ -493,7 +493,7 @@ export const PrivacyPoolABI = [
             },
             {
                 "name": "newCommitmentHash",
-                "type": "core::integer::u256",
+                "type": "core::felt252",
                 "kind": "data"
             },
             {
@@ -503,7 +503,7 @@ export const PrivacyPoolABI = [
             },
             {
                 "name": "existingNullifierHash",
-                "type": "core::integer::u256",
+                "type": "core::felt252",
                 "kind": "data"
             }
         ]
@@ -520,12 +520,12 @@ export const PrivacyPoolABI = [
             },
             {
                 "name": "label",
-                "type": "core::integer::u256",
+                "type": "core::felt252",
                 "kind": "data"
             },
             {
                 "name": "commitment",
-                "type": "core::integer::u256",
+                "type": "core::felt252",
                 "kind": "data"
             },
             {
@@ -548,7 +548,7 @@ export const PrivacyPoolABI = [
         "members": [
             {
                 "name": "tag",
-                "type": "core::integer::u256",
+                "type": "core::felt252",
                 "kind": "key"
             },
             {
@@ -558,12 +558,12 @@ export const PrivacyPoolABI = [
             },
             {
                 "name": "prevNullifierHash",
-                "type": "core::integer::u256",
+                "type": "core::felt252",
                 "kind": "data"
             },
             {
                 "name": "newCommitment",
-                "type": "core::integer::u256",
+                "type": "core::felt252",
                 "kind": "data"
             }
         ]
diff --git a/src/abis/EntryPoint.abi.ts b/src/abis/EntryPoint.abi.ts
index b7837f96a5a7d766487e9db168276dec7555f7d7..d9ed879ee3955d6815f852fb15a4d138d30da469 100644
--- a/src/abis/EntryPoint.abi.ts
+++ b/src/abis/EntryPoint.abi.ts
@@ -564,7 +564,7 @@ export const EntryPointABI = [
         },
         {
           "name": "commitment",
-          "type": "core::integer::u256",
+          "type": "core::felt252",
           "kind": "data"
         },
         {
diff --git a/src/abis/PrivacyPool.abi.ts b/src/abis/PrivacyPool.abi.ts
index 5810d7cd84c2d5b8d2d9cb7b5b7d001d7e5b6223..4975546913d69f5db13f4d3c2c54a46e75c9c481 100644
--- a/src/abis/PrivacyPool.abi.ts
+++ b/src/abis/PrivacyPool.abi.ts
@@ -206,7 +206,7 @@ export const PrivacyPoolABI = [
           "inputs": [],
           "outputs": [
             {
-              "type": "core::integer::u256"
+              "type": "core::felt252"
             }
           ],
           "state_mutability": "view"
@@ -461,12 +461,12 @@ export const PrivacyPoolABI = [
         },
         {
           "name": "label",
-          "type": "core::integer::u256",
+          "type": "core::felt252",
           "kind": "key"
         },
         {
           "name": "commitment",
-          "type": "core::integer::u256",
+          "type": "core::felt252",
           "kind": "data"
         },
         {
@@ -476,7 +476,7 @@ export const PrivacyPoolABI = [
         },
         {
           "name": "preCommitmentHash",
-          "type": "core::integer::u256",
+          "type": "core::felt252",
           "kind": "data"
         }
       ]
@@ -493,7 +493,7 @@ export const PrivacyPoolABI = [
         },
         {
           "name": "newCommitmentHash",
-          "type": "core::integer::u256",
+          "type": "core::felt252",
           "kind": "data"
         },
         {
@@ -503,7 +503,7 @@ export const PrivacyPoolABI = [
         },
         {
           "name": "existingNullifierHash",
-          "type": "core::integer::u256",
+          "type": "core::felt252",
           "kind": "data"
         }
       ]
@@ -520,12 +520,12 @@ export const PrivacyPoolABI = [
         },
         {
           "name": "label",
-          "type": "core::integer::u256",
+          "type": "core::felt252",
           "kind": "data"
         },
         {
           "name": "commitment",
-          "type": "core::integer::u256",
+          "type": "core::felt252",
           "kind": "data"
         },
         {
@@ -548,7 +548,7 @@ export const PrivacyPoolABI = [
       "members": [
         {
           "name": "tag",
-          "type": "core::integer::u256",
+          "type": "core::felt252",
           "kind": "key"
         },
         {
@@ -558,12 +558,12 @@ export const PrivacyPoolABI = [
         },
         {
           "name": "prevNullifierHash",
-          "type": "core::integer::u256",
+          "type": "core::felt252",
           "kind": "data"
         },
         {
           "name": "newCommitment",
-          "type": "core::integer::u256",
+          "type": "core::felt252",
           "kind": "data"
         }
       ]
